src/index.tsx

```diff
@@ -11,10 +10,9 @@ const createContextThief = <ContextValue extends unknown> (context: React.Contex
 	contextValue: ContextValueRef<ContextValue>;
 } => {
 	const contextValue: ContextValueRef<ContextValue> = {};
-	const ContextThief: React.FC = ({children}) => {
+	const ContextThief: React.FC<{children?: React.ReactNode | undefined}> = ({children}) => {
```

Comment:
Implicit children were removed from the React v18 types. I am using the most permissive option for this value for backwards compatibility and made children required for the test and README.md usage.